class.jetpack-gutenberg.php

```diff
@@ -937,4 +937,26 @@ public static function validate_block_embed_url( $url, $allowed = array(), $is_r
 		return false;
 	}
 
+	/**
+	 * Output an UpgradeNudge Component on the frontend of a site.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @param string $plan The plan that users need to purchase to make the block work.
+	 *
+	 * @return string
+	 */
+	public static function upgrade_nudge( $plan ) {
```

Comment:
I wonder if we should add the 2 filters we use here:
https://github.com/Automattic/jetpack/blob/557a85a6a2b0c9b5dc1535212632904255fda1e3/class.jetpack-gutenberg.php#L254-L267

Those filters are there to hide the nudges in Jetpack, I think we should use them for frontend nudges as well.